massa-grpc/src/public.rs

```diff
@@ -43,7 +43,7 @@ pub(crate) fn execute_read_only_call(
     let caller_address = match call.caller_address {
         Some(addr) => Address::from_str(&addr)?,
         None => {
-            let now = MassaTime::now()?;
```

Comment:
can the error variant corresponding to this "?" be removed as well ?

Reply:
Done